Automattic/wp-calypso
# Update Newspack Blocks

#### Changes proposed in this Pull Request

Updates Newspack Blocks to [v1.30.0](https://github.com/Automattic/newspack-blocks/releases/tag/v1.30.0).

https://github.com/Automattic/newspack-blocks/pull/725 changes how deduplication is handled.

Post Carousel also has several new options that affect how it renders on the front-end.

#### Testing instructions

1. Ensure the new version of `newspack-blocks` is synchronised with FSE
2. Create a page and insert both Blog Posts and Carousel blocks
3. Observe that the deduplication works across all blocks - no posts are duplicated
4. For the Post Carousel, observe the new options and confirm that they work as expected in the editor and on the front-end:

<img width="273" alt="Screen Shot 2021-07-19 at 10 39 28 PM" src="https://user-images.githubusercontent.com/2230142/126262925-70eb1324-acf1-4715-9ed2-663567d8ad6f.png">

  - Slide aspect ratio
  - Image fit
  - Number of slides to show at once
  - Post types (should allow posts and pages, at least, unless the site has plugins that register additional post types for the block)
